noir-projects/aztec-nr/aztec/src/keys/getters.nr

```diff
@@ -13,16 +13,34 @@ use crate::{
 global DELAY = 5;
 
 // docs:start:key-getters
-pub fn get_npk_m(context: &mut PrivateContext, address: AztecAddress) -> GrumpkinPoint {
-    get_master_key(context, address, NULLIFIER_INDEX)
+trait GetIvpkM {
```

Comment:
For the keys might makes sense just to do a single trait for these? Not as separated as the other inclusions so reduces the boilerplate a bit 🤷